scripts/release-edge.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -xe
+
+# Restore all git changes
+git restore -s@ -SW  -- packages
+
+# Bump versions to edge
+yarn jiti ./scripts/bump-edge
+
+# Resolve yarn
+YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn install
+
+# Update token
+if [[ ! -z ${NODE_AUTH_TOKEN} ]] ; then
+  echo "//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}" >> ~/.npmrc
+  echo "registry=https://registry.npmjs.org/" >> ~/.npmrc
```

Comment:
Can we store the `.npmrc` file in the root of the repo instead? I wouldn't want this to be saved on my system when accidentally (or even intentionally) running this command from the terminal.

Reply:
Planning to address this? I see that you went back and forth.